lib/task/migrate/migrations/arMigration0183.class.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Add taxonomy and terms that relate to accession events
+ *
+ * @package    AccesstoMemory
+ * @subpackage migration
+ */
+class arMigration0183
+{
+  const
+    VERSION = 183, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  public function up($configuration)
+  {
+    // Add accession event type taxonomy
+    QubitMigrate::bumpTaxonomy(QubitTaxonomy::ACCESSION_EVENT_TYPE_ID, $configuration);
+    $taxonomy = new QubitTaxonomy;
+    $taxonomy->id = QubitTaxonomy::ACCESSION_EVENT_TYPE_ID;
+    $taxonomy->parentId = QubitTaxonomy::ROOT_ID;
+    $taxonomy->sourceCulture = 'en';
+    $taxonomy->setName('Accession event type', array('culture' => 'en'));
+    $taxonomy->save();
+
+    // Add physical transfer accession event type term
+    QubitMigrate::bumpTerm(QubitTerm::ACCESSION_EVENT_PHYSICAL_TRANSFER_ID, $configuration);
+    $term = new QubitTerm;
+    $term->id = QubitTerm::ACCESSION_EVENT_PHYSICAL_TRANSFER_ID;
+    $term->parentId = QubitTerm::ROOT_ID;
+    $term->taxonomyId = QubitTaxonomy::ACCESSION_EVENT_TYPE_ID;
+    $term->sourceCulture = 'en';
+    $term->setName('Physical transfer', array('culture' => 'en'));
+    $term->save();
+
+    // Add accession event note type term
+    QubitMigrate::bumpTerm(QubitTerm::ACCESSION_EVENT_NOTE_ID, $configuration);
+    $term = new QubitTerm;
+    $term->id = QubitTerm::ACCESSION_EVENT_NOTE_ID;
+    $term->parentId = QubitTerm::ROOT_ID;
+    $term->taxonomyId = QubitTaxonomy::NOTE_TYPE_ID;
+    $term->sourceCulture = 'en';
+    $term->setName('Accession event note', array('culture' => 'en'));
+    $term->save();
+
+    return true;
+  }
+}
```

Comment:
I just thought, we're missing the new tables creation on the migration process!

Reply:
In case we need to pick this feature to another branch that is < 2.6.x, this migration perhaps should also check if these terms and DB changes exist already?

Reply:
Thanks guys... I've made the migration idempotent.